improver/calibration/rainforest_calibration.py

```diff
@@ -404,40 +402,56 @@ def _evaluate_probabilities(
         self,
         forecast_data: ndarray,
         input_data: ndarray,
+        forecast_variable: str,
+        forecast_variable_unit: str,
         output_data: ndarray,
-        preprocessing_fun: Optional[Callable[[ndarray], object]] = None,
+        model_input_converter: Optional[Callable[[ndarray], object]] = None,
     ):
         """Evaluate probability that error in forecast exceeds thresholds, setting
-        the result to 1 when `forecast + threshold <= 0`.
+        the result to 1 when `forecast + threshold` is less than or equal to
+        the lower bound of forecast_variable, as defined in constants.BOUNDS_FOR_ECDF`.
 
         Args:
             forecast_data:
                 1-d containing data for the variable to be calibrated.
             input_data:
                 2-d array of data for the feature variables of the model
+            forecast_variable:
+                name of forecast variable
+            forecast_variable_unit:
+                unit of forecast variable
             output_data:
                 array to populate with output; will be modified in place
-            preprocessing_fun:
+            model_input_converter:
                 function to convert data from ndarray to model input format,
                 only used for treelite predictor
         """
 
-        if preprocessing_fun:
-            input_dataset = preprocessing_fun(input_data)
+        if model_input_converter:
+            input_dataset = model_input_converter(input_data)
         else:
             input_dataset = input_data
 
+        bounds_data = BOUNDS_FOR_ECDF[forecast_variable]
+        bound_unit = unit.Unit(bounds_data[1])
```

Comment:
I'd suggest 'bounds_unit` to be consistent with the above variable name. I also think it would be worthwhile to unpack the lower bound into a separate variable to make the following line clearer:
```
   lower_bound = bounds_data[0][0]
```
You could also change the other instance of `lower_bound` to `lower_bound_in_fcst_units` if you wanted to be explicitly clear, but I don't think this is necessary.

Reply:
I've updated this